src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs

```diff
@@ -45,6 +45,18 @@ public class DeploymentParameters
             ServerType = serverType;
             RuntimeFlavor = runtimeFlavor;
             EnvironmentVariables["ASPNETCORE_DETAILEDERRORS"] = "true";
+
+            var config = Environment.GetEnvironmentVariable("CI");
+            if (!string.IsNullOrEmpty(config) && bool.TryParse(config, out var isCi) && isCi)
+            {
+                Configuration = "Release";
```

Comment:
This isn't always going to be a safe assumption. We are working on adding Debug CI configurations too.

Reply:
On travis & appveyor? That's where this variable is coming from.

Reply:
TeamCity also sets `CI=true`. We default to Configuration=Release when CI == true, but it's not guaranteed that CI==true implies Configuration == Release.

Reply:
Example: https://github.com/aspnet/KestrelHttpServer/pull/2523